tests/Console/ConfigurationResolverTest.php

```diff
@@ -348,14 +348,16 @@ public function testResolveConfigFileChooseFileWithPathArrayAndConfig()
     {
         $dirBase = $this->getFixtureDir();
 
-        new ConfigurationResolver(
+        $resolver = new ConfigurationResolver(
             $this->config,
             array(
                 'config' => $dirBase.'case_1/.php_cs.dist',
                 'path' => array($dirBase.'case_1/.php_cs.dist', $dirBase.'case_1/foo.php'),
             ),
             ''
         );
+
+        $this->assertInstanceOf('PhpCsFixer\Console\ConfigurationResolver', $resolver);
```

Comment:
Totally unsure what the intention of this test is, so I added a pretty much useless assertion.

Does anyone have a suggestion?

Reply:
The tests proves that when passing an array with two values as `path` options; the resolved of the configuration path is done correctly.
I agree however that the test only assures no exception is raised and not that the resolving of the path is done correctly. The added `assertInstanceOf` is a nice start though :)